### PR TITLE
fix some test setups causing import failures

### PR DESCRIPTION
**Pull request by zjgarvey**
Resolves issues <https://github.com/nod-ai/SHARK-ModelDev/issues/860> <https://github.com/nod-ai/SHARK-ModelDev/issues/859> and <https://github.com/nod-ai/SHARK-ModelDev/issues/863>

- 5 tests should newly pass with this change (the vit models listed under "no_opset_update")
- 1 test "resnetv2_50x1_bit.goog_in21k_ft_in1k_vaiq" should go from an import fail to a Numerics fail.

### CODE REVIEW

File: alt_e2eshark/onnx_tests/models/azure_models.py [hunk 1]
```diff
@@ -26,6 +27,19 @@
...
+    # TODO: unblock these when we externalize weights on import
+    # "dm_nfnet_f3.dm_in1k",
+    # "dm_nfnet_f4.dm_in1k",
+    "vit_base_r50_s16_384.orig_in21k_ft_in1k",
+    "vit_small_r26_s32_224.augreg_in21k_ft_in1k",
+    "vit_small_r26_s32_384.augreg_in21k_ft_in1k",
+    "vit_tiny_r_s16_p8_224.augreg_in21k_ft_in1k",
+    "vit_tiny_r_s16_p8_384.augreg_in21k_ft_in1k",
+]
+
```
Comment (kumardeepakamd) [inline alt_e2eshark/onnx_tests/models/azure_models.py:42]: Aren't there ~50 such models?

File: alt_e2eshark/onnx_tests/models/azure_models.py [hunk 2]
```diff
@@ -113,3 +134,38 @@ def construct_model(self):
...
+                    to_remove.append(i)
+        num_removed = 0
+        for i in to_remove:
+            _ = metadata.pop(i - num_removed)
+        mod_model_path = str(Path(self.model).parent / "model.modified.onnx")
+        onnx.save(model, mod_model_path)
+        self.model = mod_model_path
+
+
+register_test(AzureRemoveMetadataProps, "resnetv2_50x1_bit.goog_in21k_ft_in1k_vaiq")
```
Comment (kumardeepakamd) [inline alt_e2eshark/onnx_tests/models/azure_models.py:171]: LG. Wondering, what the reason for the duplicate meta data is?

**Review by kumardeepakamd — APPROVED** [review]

File: alt_e2eshark/onnx_tests/models/azure_models.py (hunk 2 above)
Comment (zjgarvey) [reply alt_e2eshark/onnx_tests/models/azure_models.py:171]: No idea. Likely an accident by someone who was quantizing it. Normally not a problem, but the onnx importer uses a model checker and it's failing there.

File: alt_e2eshark/onnx_tests/models/azure_models.py (hunk 1 above)
Comment (zjgarvey) [reply alt_e2eshark/onnx_tests/models/azure_models.py:42]: The other ones apparently did not have the problematic BatchNormalization signature encountered when updating the opset version.

**Pull request merged**